doc/components.md

```diff
@@ -264,6 +264,7 @@ final layers = images.map((image) => ParallaxLayer(await image, velocityMulitpli
 final parallaxComponent = ParallaxComponent.fromParallax(
   Parallax(
     await Future.wait(layers),
+    Vector2.all(100),
```

Comment:
We can't use the `loadParallaxComponent` here because this method receive image paths, while this example, it "loads" the layers itself, I will use game size here